integration/flink/src/main/java/io/openlineage/flink/visitor/KafkaSinkVisitor.java

```diff
@@ -0,0 +1,64 @@
+package io.openlineage.flink.visitor;
+
+import io.openlineage.client.OpenLineage;
+import io.openlineage.flink.api.OpenLineageContext;
+import java.lang.reflect.Field;
+import java.util.Collections;
+import java.util.List;
+import java.util.Properties;
+import java.util.function.Function;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.reflect.FieldUtils;
+import org.apache.flink.connector.kafka.sink.KafkaRecordSerializationSchema;
+import org.apache.flink.connector.kafka.sink.KafkaSink;
+
+@Slf4j
+public class KafkaSinkVisitor extends Visitor<OpenLineage.OutputDataset> {
+
+  public KafkaSinkVisitor(@NonNull OpenLineageContext context) {
+    super(context);
+  }
+
+  @Override
+  public boolean isDefinedAt(Object sink) {
+    return sink instanceof KafkaSink;
+  }
+
+  @Override
+  public List<OpenLineage.OutputDataset> apply(Object kafkaSink) {
+    var sink = (KafkaSink<?>) kafkaSink;
```

Comment:
seems like we don't use this variable, can be removed

Reply:
Done.